Jellyfin.Plugin.SmartPlaylist/Configuration/config.js

```
@@ -23,6 +23,174 @@
+    }
+    
+    // Fallback time formatting for older browsers
+    function formatTimeFallback(hour, minute, use12Hour) {
+        var displayMinute = minute < 10 ? '0' + minute : minute;
+        
+        if (use12Hour) {
+            var displayHour = hour === 0 ? 12 : (hour > 12 ? hour - 12 : hour);
+            var ampm = hour < 12 ? 'AM' : 'PM';
+            return displayHour + ':' + displayMinute + ' ' + ampm;
+        } else {
+            var displayHour = hour < 10 ? '0' + hour : hour;
```

Biome (2.1.2)

[error] Shouldn't redeclare 'displayHour'. Consider to delete it or rename it.

'displayHour' is defined here:


(lint/suspicious/noRedeclare)